src/check-image/check-image.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { Message } from 'discord.js';
+
+@Injectable()
+export class CheckImageService {
+  private countsDictionary: { [key: string]: number } = {};
+
+  public checkImage(message: Message): void {
+    const authorId = message.author.id;
+    if (!authorId) {
+      if (this.countsDictionary[authorId]) {
```

Comment:
Will this not always be false because of line `10`?

Reply:
Oh lol you are Right. Probably Made a mistake when refactoring

Reply:
I thought I was going crazy and had to check a few times lol also 😂